File: FastClean/FastCleanAPI/DTO's/Funcionario/FuncionarioDetails.cs
using FastCleanAPI.DTO_s.Utilizador;

namespace FastCleanAPI.DTO_s.Funcionario
{
    public class FuncionarioDetails : UtilizadorDetails
    {
        public double Preco { get; set; }

        public string TipoLimpeza { get; set; }

        public string EstadoFuncionario { get; set; }

        public double MediaReviews { get; set; }

        public string Longitude { get; set; }
        public string Latitude { get; set; }

        public string ValidadeSubscricao  { get; set; }
        public double distanciaAoCliente { get; set; }
    }
}

File: FastCleanApp/app/src/main/java/com/example/fastclean/Models/Utilizador/Funcionario/FuncionarioDetails.kt
package com.example.fastclean.Models.Utilizador.Funcionario

import com.example.fastclean.Models.Utilizador.Morada
import com.example.fastclean.Models.Utilizador.UserImage
import com.example.fastclean.Models.Utilizador.UtilizadorDetails
import java.io.Serializable

class FuncionarioDetails(
    id: Int,
    nome: String,
    val estado: String,
    val mediaReviews : Float,
    val preco : Float,
//    imagem: ByteArray,
    morada: Morada,
    latitude: Double,
    longitude: Double

): UtilizadorDetails(
    id,
    nome,
//    imagem,
    morada,
    latitude,
    longitude

), Serializable